tests/protocol/mockwiredata.py

```diff
@@ -23,76 +23,17 @@
 from azurelinuxagent.common.future import httpclient
 from azurelinuxagent.common.utils.cryptutil import CryptUtil
 
-DATA_FILE = {
-        "version_info": "wire/version_info.xml",
-        "goal_state": "wire/goal_state.xml",
-        "hosting_env": "wire/hosting_env.xml",
-        "shared_config": "wire/shared_config.xml",
-        "certs": "wire/certs.xml",
-        "ext_conf": "wire/ext_conf.xml",
-        "manifest": "wire/manifest.xml",
-        "ga_manifest": "wire/ga_manifest.xml",
-        "trans_prv": "wire/trans_prv",
-        "trans_cert": "wire/trans_cert",
-        "test_ext": "ext/sample_ext-1.3.0.zip",
-        "remote_access": None,
-        "in_vm_artifacts_profile": None
-}
+from tests.protocol.mockwiredata_filenames import *
```

Comment:
Avoid using `import *`, import only what you need.

Reply:
In this case, we actually _do_ want to import everything, though I do realize that more things might be added to that module we don't want (e.g. private implementation details, mainly). I don't think that's likely though, as that file only has data in it.

Is this an example that proves the rule? or do we still want to avoid the `import *`?

((The only real benefit to keeping the star is, I guess, readability, though maybe that's outweighed by the code smell))

Reply:
We want to be intentional with what we import. Today, this is fine, since you are using everything from that module so it's simpler to do `import *`. However, this is bad practice for several reasons, some being:
1) You don't know exactly what's imported and it's not easy to find which module this was imported from and reduces readability.
2) Let's say a module does an `import *` on this module. Then it would implicitly also import everything this module is importing, which can lead to clashes with methods defind in the current module.
3) The linter will complain about this 😄 

We've had actual issues due to this bad practice before, tests were mocking objects imported from the wrong module, etc. These two PRs did the cleanup of such code: #1704 and #1900.

Reply:
Sure, that makes sense. I'll go ahead and push a fix. Thanks!